examples/e18_webhook/src/main.rs

```diff
@@ -0,0 +1,13 @@
+use serenity::http::HttpBuilder;
+
+#[tokio::main]
+async fn main() {
+    // You don't need a token when you are only dealing with webhooks
+    let http = HttpBuilder::new("").build();
```

Comment:
```suggestion
    let http = Http::new("");
```
You don't need to use `HttpBuilder` if you're only passing in a token. Will need to change the import as well.